types/knockout/index.d.ts

```diff
@@ -139,24 +139,70 @@ interface KnockoutSubscribableStatic {
 }
 
 interface KnockoutSubscription {
+    /**
+     * Terminates a subscription 
+     */
     dispose(): void;
 }
 
 interface KnockoutSubscribable<T> extends KnockoutSubscribableFunctions<T> {
-    subscribe(callback: (newValue: T) => void, target: any, event: "beforeChange"): KnockoutSubscription;
+    /**
+     * Registers to be notified after the observable's value changes
+     * @param callback Function that is called whenever the notification happens
+     * @param target? Defines the value of 'this' in the callback function
```

Comment:
I believe we should drop the `?` from the JSDoc parameters: it's not necessary, and I'm actually seeing the question mark show up as part of the description, at least in the Typescript sandbox and VSCode.

<img width="658" alt="screen shot 2018-12-19 at 10 59 50 am" src="https://user-images.githubusercontent.com/2281166/50231869-bb51b400-037d-11e9-8f16-5aaf14ecd1d3.png">

Reply:
Done! Thank you.